README.md

```diff
@@ -67,6 +67,7 @@ To run integration tests, you will need to create a properties file like the one
     sonar.runtimeVersion=5.6
 
     orchestrator.updateCenterUrl=http://update.sonarsource.org/update-center-dev.properties
+    maven.localRepository=/home/bmurray/.m2/repository
```

Comment:
This one is required to build pass. Sonar Orchestrator returns null, if `maven.localRepository` in not set. BTW. Orchestrator should return this path as default as auto discovered.